ccmlib/cluster.py

```diff
@@ -518,8 +516,7 @@ def start(self, no_wait=False, verbose=False, wait_for_binary_proto=True,
 
                 # if the node is going to allocate_strategy_ tokens during start, then wait_for_binary_proto=True
                 node_wait_for_binary_proto = (self.can_generate_tokens() and self.use_vnodes and node.initial_token is None)
-
-                p = node.start(update_pid=False, jvm_args=jvm_args, jvm_version=jvm_version,
+                p = node.start(update_pid=True, jvm_args=jvm_args, jvm_version=jvm_version,
```

Comment:
this is tricky change... I am not 100% it is correct 
I wonder why we could start with `update_pid=False`?

Reply:
🤷